scripts/make_port_status.py

```diff
@@ -113,12 +113,21 @@ def get_mathlib4_module_commit_info(contents):
 prs = {}
 fetch_args = ['git', 'fetch', 'origin']
 nums = []
+
+sync_prs = dict()
+labels = dict()
+
 mathlib4repo = github.Github(github_token).get_repo("leanprover-community/mathlib4")
 for pr in mathlib4repo.get_pulls(state='open'):
     if pr.created_at < datetime.datetime(2022, 12, 1, 0, 0, 0):
         continue
     if 'no-source-header' in (l.name for l in pr.labels):
         continue
+    if 'mathlib3-pair' in (l.name for l in pr.labels):
+        for file in (f.filename for f in pr.get_files()):
+            sync_prs[file] = sync_prs.get(file, set()).union([pr.number])
```

Comment:
```suggestion
            sync_prs[file] = sync_prs.get(file, set()).add(pr.number)
```

Reply:
That's not quite the same. I think I need `sync_prs.get(file, set()).union([pr.number])` since `sync_prs['file']` might not exists.

Reply:
Oh, in that case you want
```suggestion
            sync_prs.setdefault(file, set()).add(pr.number)
```

Reply:
Or perhaps better, make `sync_prs` a `defaultdict`